core/src/test/java/org/apache/druid/data/input/HandlingInputRowIteratorTest.java

```diff
@@ -81,58 +87,81 @@ public void throwsExceptionWhenYieldingNext()
 
   public static class PresentRowTest
   {
+    // Create variables for tracking behaviors of mock object
+
+    boolean unsuccessfulHandlerSuccessful;
+
+    // Create variables for tracking behaviors of mock object
+
+    boolean successfulHandlerSuccessful;
+
     private static final InputRow INPUT_ROW1 = EasyMock.mock(InputRow.class);
     private static final InputRow INPUT_ROW2 = EasyMock.mock(InputRow.class);
```

Comment:
Should we be using EasyMock and mockito in the same class? This seems kinda confusing. My personal preference is mockito, but I'd be ok if you chose to standardize on EasyMock too. As long as there is only one mocking system per test class.

Reply:
Great suggestion! I also prefer Mockito, I will change EasyMock mock to Mockito mock.